vpr/src/base/clustered_netlist_utils.h

```diff
@@ -27,4 +27,15 @@ class ClusteredPinAtomPinsLookup {
     vtr::vector<AtomPinId, ClusterPinId> atom_pin_connected_cluster_pin_;
 };
 
+class ClusterAtomsLookup {
```

Comment:
comment what this is for

Reply:
what the private data is, automatically intialized, order 1 lookups, comment data

Reply:
state that it should only be called after clustered netlist is created